src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -17,99 +18,102 @@
         }
     }
 
     @Override
     public List<Manufacturer> getAll() {
-        String query = "SELECT * FROM manufacturers WHERE deleted = FALSE";
+        String getAllRequest = "SELECT * FROM manufacturers WHERE is_deleted = false";
+        List<Manufacturer> allManufacturers = new ArrayList<>();
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
-            List<Manufacturer> manufacturers = new ArrayList<>();
-            ResultSet resultSet = preparedStatement.executeQuery();
+                Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet =
+                    getAllStatement.executeQuery(getAllRequest);
             while (resultSet.next()) {
-                manufacturers.add(getManufacturer(resultSet));
+                allManufacturers.add(setValuesFromDB(resultSet));
             }
-            return manufacturers;
-        } catch (SQLException throwable) {
-            throw new DataProcessingException("Couldn't get a list of manufacturers "
-                    + "from manufacturers table. ",
-                    throwable);
+            return allManufacturers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all manufacturer from DB", e);
         }
     }
 
     @Override
     public Manufacturer update(Manufacturer manufacturer) {
-        String query = "UPDATE manufacturers SET manufacturer_name = ?, manufacturer_country = ?"
-                + " WHERE manufacturer_id = ? AND deleted = FALSE";
+        String updateRequest = "UPDATE manufacturers SET name = ?, "
+                + "country = ? WHERE id = ? and is_deleted = false;";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement preparedStatement
-                        = connection.prepareStatement(query)) {
-            preparedStatement.setString(1, manufacturer.getName());
-            preparedStatement.setString(2, manufacturer.getCountry());
-            preparedStatement.setLong(3, manufacturer.getId());
-            preparedStatement.executeUpdate();
-            return manufacturer;
-        } catch (SQLException throwable) {
-            throw new DataProcessingException("Couldn't update a manufacturer "
-                    + manufacturer + " ", throwable);
+                PreparedStatement createStatement =
+                        connection.prepareStatement(updateRequest)) {
+            createStatement.setString(1, manufacturer.getName());
+            createStatement.setString(2, manufacturer.getCountry());
+            createStatement.setLong(3, manufacturer.getId());
+            if (createStatement.executeUpdate() > 0) {
+                return manufacturer;
+            }
+            throw new RuntimeException("No such manufacturer " + manufacturer.getId());
```

Comment:
if you created a suitable constructor for DataProcessingException, let's use it here as well

Reply:
++